data/transition-sites/walesoffice.yml

```diff
@@ -1,8 +1,8 @@
 ---
 site: walesoffice
-whitehall_slug: wales-office
+whitehall_slug: office-of-the-secretary-of-state-for-wales
 host: www.walesoffice.gov.uk
 tna_timestamp: 20130129084240
-homepage_furl: www.gov.uk/wales-office
-homepage: https://www.gov.uk/government/organisations/wales-office
-css: wales-office
+homepage_furl: www.gov.uk/office-of-the-secretary-of-state-for-wales
+homepage: https://www.gov.uk/government/organisations/office-of-the-secretary-of-state-for-wales
+css: office-of-the-secretary-of-state-for-wales
```

Comment:
The CSS in Bouncer is still [wales-office](https://github.com/alphagov/bouncer/blob/24d7a7b0b392465b28da2ca80ab140c83c0020c8/public/gone.css#L374-L381) so we can leave it as that for the moment